README.md

```diff
@@ -43,21 +43,53 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
 ### Developer's Build 
 ```sh
 $./autogen.sh
-
 $./configure --prefix=<ucc-install-path> --with-ucx=<ucx-install-path>
```

Comment:
no space between "$" and command, maybe add it since changing readme file anyway?